ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,196 @@
+        ]
+    }
+
+    private func labelContainerLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            labelContainer.widthAnchor.constraint(equalToConstant: Constants.contactWidth),
+            labelContainer.centerXAnchor.constraint(equalTo: centerXAnchor),
+            labelContainer.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndLabelContainer),
+            labelContainer.leadingAnchor.constraint(equalTo: leadingAnchor),
+            labelContainer.trailingAnchor.constraint(equalTo: trailingAnchor),
+            labelContainer.bottomAnchor.constraint(equalTo: bottomAnchor),
+            labelContainer.heightAnchor.constraint(greaterThanOrEqualToConstant: 2.0 * Constants.labelMinimumHeight)
+        ]
+    }
+
+    private func titleLabelLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            titleLabel.topAnchor.constraint(equalTo: labelContainer.topAnchor),
+            titleLabel.leadingAnchor.constraint(equalTo: labelContainer.leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: labelContainer.trailingAnchor),
+            titleLabel.centerXAnchor.constraint(equalTo: labelContainer.centerXAnchor),
+            titleLabel.widthAnchor.constraint(equalTo: labelContainer.widthAnchor),
+            titleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.labelMinimumHeight)
+        ]
+    }
+
+    private func subtitleLabelLayoutConstraints() -> [NSLayoutConstraint] {
+        guard let subtitleLabel = subtitleLabel else {
+            return []
+        }
+
+        return [
+            subtitleLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor),
+            subtitleLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
+            subtitleLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
+            subtitleLabel.centerXAnchor.constraint(equalTo: centerXAnchor),
+            subtitleLabel.widthAnchor.constraint(equalTo: widthAnchor),
+            subtitleLabel.bottomAnchor.constraint(equalTo: bottomAnchor),
+            subtitleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.labelMinimumHeight)
+        ]
+    }
+
+    private func identifierLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            titleLabel.widthAnchor.constraint(equalTo: labelContainer.widthAnchor),
+            titleLabel.centerXAnchor.constraint(equalTo: labelContainer.centerXAnchor),
+            titleLabel.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndLabelContainer),
+            titleLabel.leadingAnchor.constraint(equalTo: labelContainer.leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: labelContainer.trailingAnchor),
+            titleLabel.bottomAnchor.constraint(equalTo: labelContainer.bottomAnchor)
+        ]
+    }
+
+    private func setupTitleLabel(using firstName: String) {
```

Comment:
>firstName [](start = 39, length = 9)

title

Reply:
Fixed this along with `setupSubtitleLabel()` accordingly.